example/raycast.js

```diff
@@ -177,11 +181,16 @@ function addRaycaster() {
 
 			hitMesh.position.set( pointDist - length, 0, 0 );
 
-			cylinderMesh.position.set( pointDist - ( length / 2 ), 0, 0 );
-			cylinderMesh.scale.set( 1, length, 1 );
+			const lineLength = res.length ? length - raycaster.near : length - raycaster.near - ( pointDist - raycaster.far );
+
+			cylinderMesh.position.set( pointDist - raycaster.near - ( lineLength / 2 ), 0, 0 );
+			cylinderMesh.scale.set( 1, lineLength, 1 );
 
 			cylinderMesh.rotation.z = Math.PI / 2;
 
+			hitMesh.visible = res.length > 0;
+			origMesh.visible = res.length > 0;
```

Comment:
Lets keep `origMesh` visible regardless of whether or not we see a hit. It's a little confusing to see those sphere flicker, I think